src/services/contact-notifier.js

```diff
@@ -32,18 +118,66 @@ module.exports = {
     return sgMail.send(payload);
   },
 
-  async sendInternalCampaignCreated({ campaign }) {
-    const html = await emailTemplates.render('internal/campaign.created', { campaign });
+  async scheduleCampaignCreated({ campaignId }) {
+    const campaign = await Campaign.findOne({ _id: campaignId });
     const advertiser = await AdvertiserRepo.findById(campaign.get('advertiserId'));
+    const materialCollectUri = await campaign.get('vMaterialCollectUri');
+    const html = await emailTemplates.render('campaign.created', { campaign, materialCollectUri });
     const subject = `A new campaign was created for ${advertiser.name}`;
-    const to = await this.resolveAddresses(campaign.get('notify.internal'));
-    return this.send({ to, subject, html });
+    const to = await this.resolveAddresses(campaign.get('notify.external'));
+    const cc = await this.resolveAddresses(campaign.get('notify.internal'));
```

Comment:
All these functions seem pretty repetitive... any way we can clean them up / consolidate?